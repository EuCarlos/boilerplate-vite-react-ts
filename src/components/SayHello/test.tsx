import { render, screen } from '@testing-library/react'
import { expect, it } from 'vitest'
import { SayHello } from '.'

it('should render Heading with text Hello YouTube', () => {
  render(<SayHello>Hello Test!</SayHello>);
  const heading = screen.getByRole('heading', { name: /Hello Test/ });
  expect(heading).toBeInTheDocument();
  expect(heading).toMatchSnapshot();
});
