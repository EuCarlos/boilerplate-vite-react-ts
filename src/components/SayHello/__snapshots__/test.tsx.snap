// Vitest Snapshot v1

exports[`should render Heading with text Hello YouTube 1`] = `
<h1>
  Hello Test!
</h1>
`;
